debian/bullseye/8.0/Dockerfile

```diff
@@ -126,8 +126,8 @@ RUN curl -L "https://download.newrelic.com/php_agent/archive/${NEWRELIC_VERSION}
   && ln -s /lib64/ld-linux-x86-64.so.2 /lib/ld-linux-x86-64.so.2
 
 # Tideways
-RUN bash -c 'if [ "${TARGETPLATFORM}" == "linux/amd64" ]; then echo "deb http://s3-eu-west-1.amazonaws.com/tideways/packages debian main" > /etc/apt/sources.list.d/tideways.list && \
-   curl -sS "https://s3-eu-west-1.amazonaws.com/tideways/packages/EEB5E8F4.gpg" | apt-key add - && \
+RUN bash -c 'if [ "${TARGETPLATFORM}" == "linux/amd64" ]; then echo "deb https://packages.tideways.com/apt-packages-main any-version main" > /etc/apt/sources.list.d/tideways.list && \
```

Comment:
Hey, 
sorry, but I have some concerns about this line.
Did you build it locally?

Reply:
Hey Alexander,

What are your concerns here? Happy to answer all questions you might have.

We recommend our customers to build this in a similar way: https://support.tideways.com/documentation/setup/installation/docker-with-compose.html#tideways-daemon-dockerized

Are you concerned about the `any-version` ? We don't publish specific packages for debian/ubuntu (and never have) or for specific releases as the `deb`s work on all of these systems.

As nothing else changed here except the URLs and `debian` -> `any-version` I'm not sure what else you could be referring too; sorry If I'm missing something here!

> Did you build it locally?

Yes. I had to work around https://download.newrelic.com/php_agent/archive/9.19.0.309/newrelic-php5-9.19.0.309-linux.tar.gz not being available but I recall no other errors.

Reply:
I did another run and I see what you're referring too now.

The key adding fails as curl doesn't follow the, new, redirect here. Sorry for not thinking of that when we created a redirect and thank you very much for catching it!